src/solver/hydro/management/management.cpp

```diff
@@ -290,43 +290,31 @@ bool HydroManagement::checkWeeklyMinGeneration(uint tsIndex, Data::Area& area) c
     return true;
 }
 
-bool HydroManagement::checkHourlyMinGeneration(uint tsIndex, Data::Area& area) const
+bool HydroManagement::checkHourlyMinMaxGeneration(uint tsIndex, uint tsIndexPowerCredits, Data::Area& area) const
```

Comment:
**checkHourlyMinMaxGeneration** : **checkGenerationPowerConsistency** is more accurate

Reply:
Done!